examples/searching.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
I only added some cells for playing with `search_things`

Reply:
Yeah, ipynb files never play nice with git diff...